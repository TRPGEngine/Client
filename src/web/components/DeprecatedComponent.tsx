import { TMemo } from '@shared/components/TMemo';
import { t } from '@shared/i18n';
import { Alert } from 'antd';
import React from 'react';

export const DeprecatedComponent: React.FC = TMemo(() => {
  return (
    <Alert
      type="error"
      message={t('该组件已被弃用, 如果影响到您的使用请联系开发者')}
    />
  );
});
DeprecatedComponent.displayName = 'DeprecatedComponent';
