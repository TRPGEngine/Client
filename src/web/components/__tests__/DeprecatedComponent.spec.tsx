import { render } from '@testing-library/react';
import React from 'react';
import { DeprecatedComponent } from '../DeprecatedComponent';

describe('DeprecatedComponent', () => {
  test('render', () => {
    const wrapper = render(<DeprecatedComponent />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
