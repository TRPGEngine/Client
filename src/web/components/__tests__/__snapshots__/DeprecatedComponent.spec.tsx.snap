// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeprecatedComponent render 1`] = `
<div>
  <div
    class="ant-alert ant-alert-error ant-alert-no-icon"
    data-show="true"
    role="alert"
  >
    <div
      class="ant-alert-content"
    >
      <div
        class="ant-alert-message"
      >
        该组件已被弃用, 如果影响到您的使用请联系开发者
      </div>
      <div
        class="ant-alert-description"
      />
    </div>
  </div>
</div>
`;
